k8s/chart/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicas: 1
+image:
+  registry: andrewstuart
```

Comment:
Unsurprisingly, I think @mmoayyed (presumably, since cas is the only image) already has [an apereo Docker hub group](https://hub.docker.com/r/apereo/cas/) set up. So it's probably just a matter of getting access For this build process, using Makefile for orchestration, I don't think we can quite piggyback on the automated Docker hub builds. AFAIK that essentially requires you to perform the build inside the Dockerfile, which means complicating the build to the point of either using a base image that has all the tools for the build, or installing all the tools and cleaning up in a single `RUN` command in order to avoid dragging all the build tools into the runtime environment.

It probably wouldn't be too time consuming to set up Travis CI with permissions to push to the Apereo group on Docker hub, and at that point we just need to alter the script to additionally call the appropriate make targets so that the image gets pushed automatically.